ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Runner.scala

```diff
@@ -85,15 +88,14 @@ class Runner(name: String, construct: ParticipantId => KeyValueLedger) {
   private def startIndexerServer(
       config: Config,
       readService: ReadService,
-  )(implicit executionContext: ExecutionContext): Resource[Unit] =
+  )(implicit executionContext: ExecutionContext, ctx: LoggingContext): Resource[Unit] =
```

Comment:
I'm not keen on `ctx` as the name for the logging context. There's another context on the same line in this particular case. Can we spell it out, or at least go with `logCtx`?

Reply:
Yes, I had the same thought while making this, I'll definitely make this part of the finished PR, thanks.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/4097/commits/23a3abb3c4026ca85e05491b6ccd5f3d422484b4